pkg/maps/worldcidrsmap/worldcidrs.go

```diff
@@ -114,11 +129,15 @@ func (m *worldCIDRsMap) Add(cidrs ...netip.Prefix) error {
 	vals := make([]WorldCIDRVal, 0, len(cidrs))
 
 	for _, cidr := range cidrs {
+		// TODO: ipv6 support
+		if cidr.Addr().Is6() {
```

Comment:
should we handle/log this case further up the stack?

Reply:
Yeah, we should probably warn in the parsing code.

Reply:
yeah, good idea.